src/main/java/frc/robot/Constants.java

```diff
@@ -28,4 +28,13 @@
     public static final int kNavigatorPort = 0;
     public static final int kOperatorPort = 1;
   }
+
+  public static final class IntakeConstants {
+    public static final int kLeftForward = 0;
+    public static final int kLeftReverse = 1;
```

Comment:
It might be better to just make an array called kIntakeSolenoidPorts with 0, and 1 stored. Then access the array port numbers